modules/xfem/src/kernels/ConcentrationDiffusion.C

```diff
@@ -0,0 +1,52 @@
+/****************************************************************/
```

Comment:
There's the MatDiffusion model in the phase_module that is very similar to this.  Maybe we could use that (after adding the base_name option to it) rather than adding another Diffusion kernel.  It should probably also be moved out of the phase_field module.